sqlglot/generator.py

```diff
@@ -2083,9 +2083,22 @@ def query_modifiers(self, expression: exp.Expression, *sqls: str) -> str:
             self.sql(expression, "order"),
             *offset_limit_modifiers,
             *self.after_limit_modifiers(expression),
+            *self.sql(expression, "option"),
             sep="",
         )
 
+    def option_sql(self, expression: exp.Option):
+        option_type = self.sql(expression, "option_type")
+        option_value = self.sql(expression, "option_value")
+        if option_value:
+            optional_equal_sign = "= " if expression.args.get("requires_equals") else ""
+            return f"{option_type} {optional_equal_sign}{option_value}"
+        return option_type
+
+    def options_sql(self, expression: exp.Options):
+        options_sql = csv(*[self.sql(opt) for opt in expression.args.get("options", [])])
+        return f" OPTION({options_sql})"
```

Comment:
This should be irrelevant after refactoring, but when you need a csv in the future you can do:

```python
self.expressions(expression, key="options")
```

Reply:
Sure - thanks!